arangod/Replication2/StateMachines/Prototype/PrototypeStateMethods.cpp

```diff
@@ -595,14 +595,14 @@ auto PrototypeStateMethods::createInstance(TRI_vocbase_t& vocbase)
 [[nodiscard]] auto PrototypeStateMethods::get(LogId id, std::string key,
                                               LogIndex waitForApplied) const
     -> futures::Future<ResultT<std::optional<std::string>>> {
-  return get(id, std::move(key), {.waitForApplied = waitForApplied});
+  return get(id, std::move(key), {waitForApplied, std::nullopt});
```

Comment:
It doesn't compile under GCC 11, 12.
I don't think it don't write nullopt explicit is really important, so I changed this